<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestSuiteExecutiveDialog</class>
 <widget class="QDialog" name="TestSuiteExecutiveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Suite Executive</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="6">
    <widget class="QLineEdit" name="testSuiteFileName">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Test Suite XML File:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Test Suite Progress:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="6">
    <widget class="QProgressBar" name="suiteProgress">
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>%v/%m tests</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QLineEdit" name="testROM">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Test ROM:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <widget class="QPushButton" name="abort">
     <property name="text">
      <string>Abort</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="executeSelection">
     <property name="text">
      <string>Execute Selection</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="execute">
     <property name="text">
      <string>Execute All</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="clear">
     <property name="text">
      <string>Clear Results</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="load">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="2" column="6">
    <widget class="QCheckBox" name="recordInputs">
     <property name="text">
      <string>Record Inputs</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Test Progress:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QProgressBar" name="testProgress">
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string>%v/%m frames</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Pass:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QProgressBar" name="passRate">
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string>%p%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
